dokku

```diff
@@ -100,13 +100,10 @@ execute_dokku_cmd() {
     nginx|nginx:*)
       local PLUGIN_NAME=${PLUGIN_NAME/nginx/nginx-vhosts}
       ;;
-    receive|deploy|build|release)
+    deploy|delete|ls|run|cleanup)
```

Comment:
Thats fair. I guess I've been thinking of our minors as majors until we do the big 1.0.0. But I don't care either way, given that nothing official has called this stuff (afaik).